src/struct.ts

```diff
@@ -124,6 +125,35 @@ export class Struct<Type = unknown, Schema = unknown> {
   }
 }
 
+// String instead of a Symbol in case of multiple different versions of this library.
+const ExactOptionalBrand = 'EXACT_OPTIONAL';
+
+/**
+ * An `ExactOptionalStruct` is a `Struct` that is used to create exactly optional
+ * properties of `object()` structs.
+ */
+export class ExactOptionalStruct<
+  Type = unknown,
+  Schema = unknown,
+> extends Struct<Type, Schema> {
+  // eslint-disable-next-line no-restricted-syntax
```

Comment:
Can you add a comment to clarify why this is necessary?

Reply:
bc66cf8